setup.py

```diff
@@ -19,11 +20,21 @@
 DOWNLOAD_URL = 'http://github.com/jakevdp/supersmoother'
 LICENSE = 'BSD 3-clause'
 
-import supersmoother
-VERSION = supersmoother.__version__
+
```

Comment:
This function removes the supersmoother dependency from setup.py which makes it impossible to install the project unless your system already has numpy installed